worker/mutation.go

```diff
@@ -424,6 +424,32 @@ func checkSchema(s *pb.SchemaUpdate) error {
 			x.ParseAttr(s.Predicate))
 	}
 
+	if s.Unique {
```

Comment:
a switch statement would look better here. and add a todo for the types that are added here.